cmd/serve.go

```diff
@@ -100,6 +101,14 @@ func serve() {
 		logWithCommand.Info("state validator disabled")
 	}
 
+	if serverConfig.WatchedAddressGapFillerEnabled {
+		service := fill.New(serverConfig)
```

Comment:
Should the service have some type of shutdown procedure in response to the interrupt notify below?

Reply:
Graceful shutdown implemented.